kamon/README.md

```diff
@@ -0,0 +1,143 @@
+# Kamon 
+
+New Relic [provides two reporters for Kamon telemetry](https://github.com/newrelic/kamon-newrelic-reporter):
```

Comment:
also add in a link to kamon itself, please.

Reply:
good call!